modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/executor/MethodArgumentResolverDelegate.java

```diff
@@ -50,6 +53,8 @@
     private final Method method;
     private final JavaTypeLoader typeLoader = new JavaTypeLoader(this.getClass().getClassLoader());
     private ArgumentResolver<? extends Object>[] argumentResolvers;
+    private Map<java.lang.reflect.Parameter, ParameterGroupArgumentResolver<? extends Object>> parameterGroupResolvers;
+    private boolean initialized = false;
```

Comment:
I did this lazy initialization because when creating the ResolverDelegate I don't have the operation model to ask for the ParameterGroupModelProperty and create the associated resolvers